### PR TITLE
Updating task table to leverage col definition

### PR DESCRIPTION
**Pull request by mlunoe**
And fixing some issues with the current usage
Before:

After:

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @orlandohohmeier

File: src/js/components/TaskTable.js [hunk 1]
```diff
@@ -21,6 +21,8 @@ const METHODS_TO_BIND = [
   'renderVersion'
 ];
 
+const LEFT_ALIGN_PROPS = ['cpus', 'disk', 'log', 'mem', 'version'];
```
Comment (wavesoft) [inline src/js/components/TaskTable.js]: Do you mean `RIGHT_ALIGN_PROPS` ?

Comment (mlunoe) [inline src/js/components/TaskTable.js]: Right, hehe. Copy/paste strikes again!
Done

**Comment by jfurrow** [comment]
LGTM :ship:

**Comment by mlunoe** [comment]
**mesosphere-ci** Retest this please

**Pull request merged**